local-cli/nuget-exe/package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "nuget-exe",
+    "version": "0.1.0",
+    "description": "Nuget.exe wrapper",
+    "license": "Apache License 2.0",
```

Comment:
on nuget.org, the nuget.core package has an apache license, so I made included the apache license 2.0 and credited the apache foundation in the README. We should check into whether or not this is actually correct.